racon/1.5.0/README.md

```diff
@@ -1,23 +1,46 @@
-# racon container
+# Racon + Minimap2 Container
 
-Main tool : [racon](https://github.com/lbcb-sci/racon)
+This container combines [Racon](https://github.com/lbcb-sci/racon) and [Minimap2](https://github.com/lh3/minimap2) for efficient polishing and alignment in long-read assembly workflows.
 
-Code repository: https://github.com/lbcb-sci/racon
+## Tools Included
+- **Racon**: Polishes long-read assemblies.
+- **Minimap2**: Generates alignments for use with Racon and other polishing tools.
 
-Basic information on how to use this tool:
-- executable: racon
-- help: -h
-- version: -v
-- description: Polishes long read assemblies
+### Code Repositories
+- Racon: [GitHub Repository](https://github.com/lbcb-sci/racon)
```

Comment:
I think this file is in the wrong directory. Should this be in the 1.5.0-minimap2 subdirectory?

Reply:
you are correct updated and reset the README for 1.5.0 to its correct README